ppl/src/test/java/org/opensearch/sql/ppl/parser/AstBuilderTest.java

```diff
@@ -607,6 +609,26 @@ public void testParseCommand() {
         ));
   }
 
+  @Test
+  public void testStringLiteralHighlightCommand() {
+    assertEqual("source=t | highlight('FieldA')",
```

Comment:
can double quotes be used in the highlight parameter? If so, should it be tested?

Reply:
Yes double quotes in highlight parameter are supported. Both single and double quoted parameters will be interpreted as string literals at this parsing level. I don't think it's overly necessary to add a test for it at this level.